File: hw/toapanta/u2/hw19inherence/javaScriptFarmSystem/model/Cow.js
<<<<<<< HEAD
const FarmAnimal = require('./FarmAnimal');

class Cow extends FarmAnimal {
    constructor(isProducingMilk, litersADay, id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location) {
        super(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location);
        this.isProducingMilk = isProducingMilk;
        this.litersADay = litersADay;
    }

    toString() {
        return `Cow{isProducingMilk=${this.isProducingMilk}, litersADay=${this.litersADay}, ${super.toString()}}`;
    }

    isIsProducingMilk() {
        return this.isProducingMilk;
    }

    setIsProducingMilk(isProducingMilk) {
        this.isProducingMilk = isProducingMilk;
    }

    getLitersADay() {
        return this.litersADay;
    }

    setLitersADay(litersADay) {
        this.litersADay = litersADay;
    }
}

=======
const FarmAnimal = require('./FarmAnimal');

class Cow extends FarmAnimal {
    constructor(isProducingMilk, litersADay, id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location) {
        super(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location);
        this.isProducingMilk = isProducingMilk;
        this.litersADay = litersADay;
    }

    toString() {
        return `Cow{isProducingMilk=${this.isProducingMilk}, litersADay=${this.litersADay}, ${super.toString()}}`;
    }

    isIsProducingMilk() {
        return this.isProducingMilk;
    }

    setIsProducingMilk(isProducingMilk) {
        this.isProducingMilk = isProducingMilk;
    }

    getLitersADay() {
        return this.litersADay;
    }

    setLitersADay(litersADay) {
        this.litersADay = litersADay;
    }
}

>>>>>>> 203b676d8f105a34d549ec251a59c11aa2c57532
module.exports = Cow;

File: hw/toapanta/u2/hw19inherence/javaFarmSystem/src/main/java/ec/edu/espe/farmsystem/model/Cow.java
<<<<<<< HEAD
package ec.edu.espe.farmsystem.model;
import java.util.Date;

/**
 *
 * @author Joseph B. Medina
 */
public class Cow extends FarmAnimal{
    
    private boolean isProducingMilk;
    private float litersADay;

    public Cow(boolean isProducingMilk, float litersADay, int id, String breed, Date bornOn, String gendeR, boolean isAbleToReproduce, float weight, Cage cage, Location location) {
        super(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location);
        this.isProducingMilk = isProducingMilk;
        this.litersADay = litersADay;
    }

    public boolean isIsProducingMilk() {
        return isProducingMilk;
    }

    public void setIsProducingMilk(boolean isProducingMilk) {
        this.isProducingMilk = isProducingMilk;
    }

    public float getLitersADay() {
        return litersADay;
    }

    public void setLitersADay(float litersADay) {
        this.litersADay = litersADay;
    }

    @Override
    public String toString() { 
        return "Cow{" + "isProducingMilk=" + isProducingMilk + ", litersADay=" + litersADay + ", " + super.toString() + '}';
    }
    
}
=======
package ec.edu.espe.farmsystem.model;
import java.util.Date;

/**
 *
 * @author Joseph B. Medina
 */
public class Cow extends FarmAnimal{
    
    private boolean isProducingMilk;
    private float litersADay;

    public Cow(boolean isProducingMilk, float litersADay, int id, String breed, Date bornOn, String gendeR, boolean isAbleToReproduce, float weight, Cage cage, Location location) {
        super(id, breed, bornOn, gendeR, isAbleToReproduce, weight, cage, location);
        this.isProducingMilk = isProducingMilk;
        this.litersADay = litersADay;
    }

    public boolean isIsProducingMilk() {
        return isProducingMilk;
    }

    public void setIsProducingMilk(boolean isProducingMilk) {
        this.isProducingMilk = isProducingMilk;
    }

    public float getLitersADay() {
        return litersADay;
    }

    public void setLitersADay(float litersADay) {
        this.litersADay = litersADay;
    }

    @Override
    public String toString() { 
        return "Cow{" + "isProducingMilk=" + isProducingMilk + ", litersADay=" + litersADay + ", " + super.toString() + '}';
    }
    
}
>>>>>>> 203b676d8f105a34d549ec251a59c11aa2c57532
